README.docker.md

```diff
@@ -0,0 +1,26 @@
+# Running in Docker
+
+## Setup
+
+Install docker and docker-compose
+
+Then setup your environment variables:
+
+    sed 's/export //' env.test > env.docker
+
+## Running
+
+    docker-compose build
```

Comment:
This has to do with the docker mac setup.

My guess is you have docker installed, but the terminal you ran this in does not know about it. I think on the mac, you need to open a docker terminal where you would run these commands.

If that is not the case then perhaps the docker install did not work since it seems to be looking for a VM where it should have put it.